src/scenes/Event/components/EventInfo/styles.js

```diff
@@ -10,7 +10,8 @@ const styles = (theme) => ({
     textOverflow: 'ellipsis',
```

Comment:
remove

Reply:
wont this just cutoff the end of the text and some of the letters will still be missing?

Reply:
well if we remove textOverflow: 'ellipsis', whiteSpace: 'nowrap',overflow: 'hidden', and add "word-wrap:break-word;". If the text is more than the div width, it will just be two lines

Reply:
i'm not sure if we should show a qtum address on two lines. think that feels kind of confusing for the user. can we adjust the grid of the main content to give the side content more room?

Reply:
but the thing is, no matter how you adjust that, if the user drags the window into a smaller size, you'll have the same problem.

Reply:
true, change made.